documentation/sphinx/source/release-notes/release-notes-7130.rst

```diff
@@ -0,0 +1,62 @@
+.. _release-notes:
+
+#############
+Release Notes
+#############
+
+71.3.0
+======
+
+Features
+--------
+- Added MonotonicTime field, based on system clock, to CommitDebug trace events, for accurate timing.
+- BlobGranule driven Backup + restore
+- Encryption at-Rest, supports per-tenant and/or per-cluster encryption semantics
+- External Vault support for per-tenant blob granule storage location
+- Improved APIs for creating and managing Blob Granule ranges
+- Updated tenants to be ID oriented. Tenants cannot be upgraded from previous releases.
+- Support for adding restored data clusters back to a metacluster and recovering a management cluster from the data clusters
+- Add support for locking tenants
+- Added a new function fdb_database_get_client_status providing a client-side connection status information in json format.
+- Added new database options to define defaults for transaction options report_conflicting_keys and used_during_commit_protection_disable.
+- Added a new network option retain_client_library_copies to avoid deleting the temporary library copies after completion of the process. This may be useful in various debugging and profiling scenarios.
+- Added a new network option trace_initialize_on_setup to enable client traces already on fdb_setup_network, so that traces do not get lost on client configuration issues
```

Comment:
```suggestion
- Added a new network option trace_initialize_on_setup to enable client traces already on fdb_setup_network, so that traces do not get lost on client configuration issues.
```